routes/conditions/conditions.njk

```diff
@@ -2,27 +2,46 @@
 
 {% block content %}
 
-    <h1>{{ __('conditions.title') }}<br/> <span class="text-base">{{ __('conditions.patient')}} {{ patientName(data) }}</span></h1>
+    <h1 class="border-gray-400 border-b">{{ __('conditions.title') }}<br/> <span class="text-base">{{ __('conditions.patient')}} {{ patientName(data) }}</span></h1>
 
-    <div>
-        <p>
-            {{ __('conditions.intro') }}
-        </p>
-
-        <ol>
-        {% for cond in data.conditions %}
-            <li>{{ cond.name_of_condition }}</li>
+    <div class="mt-10">
+        <h2 class="text-2xl mb-4">{{ __('conditions.medical_conditions') }}</h2>
+        
+        {% for condition in data.conditions %}  
+            <div class="flex border-t border-b">
+                <div class="w-1/2">
+                    <p>{{ condition.name_of_condition }}</p>
+                    <p class="text-base">{{ __('conditions.icd_code') }} XXXX</p>
+                    <p class="text-base">{{ __('conditions.symptoms_began') }} {{ condition.symptoms_began }}</p>
+                </div>
+                <div class="w-1/2">
+                    <!-- <a href="/en/conditions/{{ loop.index }}/edit">Edit condition</a><br> -->
+                    <form action="/en/conditions/{{ loop.index }}/delete" method="POST">
+                        <input type="hidden" name="_csrf" value="{{ csrfToken }}">
+                        <a href="#" onclick="this.parentNode.submit();" class="text-sm">
+                            <svg class="w-4 h-4 inline fill-current mr-2" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 20 20"><path d="M10 1.6a8.4 8.4 0 100 16.8 8.4 8.4 0 000-16.8zm5 9.4H5V9h10v2z"/></svg>
+                            {{ __('conditions.remove_condition' )}}
+                        </a>
```

Comment:
From an accessibility point of view this should semantically be a button so either should be a `<button>` tag or should be marked up as a button using ARIA (link to mdn ARIA role=button)

Maybe @JuliannaR might want to weigh in on this? Since we _might_ need to style it as a button as well, but maybe not I'm not 100% sure

## Resources
http://web-accessibility.carnegiemuseums.org/content/buttons/
https://a11y-101.com/design/button-vs-link
https://www.webaxe.org/proper-use-buttons-links/

Reply:
added the role ... we'll revisit this all later